src/Command/NamespaceToPSR4Command.php

```diff
@@ -49,7 +49,8 @@ protected function configure(): void
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $path = (string) $input->getArgument('path');
-        $namespaceRoot = (string) $input->getOption('namespace-root');
+        $namespaceRoot = rtrim((string) $input->getOption('namespace-root'), '\\');
+        $namespaceRoot = str_replace('\\\\', '\\', $namespaceRoot);
```

Comment:
This fixes the issue where it incorrectly states it will transform `App\Tests\Util` to `App\\Tests\Util`, which doesn't result in a file change.